spec/requests/api/v1/weathers_controller_spec.rb

```diff
@@ -31,13 +32,17 @@
     end
 
     it 'returns humidity to be nil' do
-      expect(JSON.parse(response.body)['humidity']).to be_nil
+      expect(JSON.parse(response.body)['relativehumidity_2m']).to be_nil
```

Comment:
do tests about what you recieve and not of what you dont recieve

Reply:
like if the front end team recieves extra info we dont mind, we need for them to recieve what they need